### ISSUE TITLE
(missing: SQLITE3_LIBRARIES

### ISSUE THREAD

**Issue by vectorez** [nextcloud/client-building#116]
-- Using Qt  ()
CMake Error at C:/Program Files/CMake/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find SQLite3 (missing: SQLITE3_LIBRARIES) (Required is at least
  version "3.8.0")
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindSQLite3.cmake:56 (find_package_handle_standard_args)
  src/csync/CMakeLists.txt:22 (find_package)
please help me

**Comment by allexzander** [comment]
@vectorez Yes, you ned to have SQLite built and installed on your dev machine. You can get these pre-built binaries if you have problems with building it locally https://download.nextcloud.com/desktop/development/Windows/Dependencies/Craft64.zip
